esx_service/utils/kvESX.py

```diff
@@ -31,6 +31,7 @@
 
 # VSphere lib to access ESX proprietary APIs.
 DISK_LIB = "/lib/libvmsnapshot.so"
+DISK_LIB64 = "/lib64/libvmsnapshot.so"
```

Comment:
Both 6.0 and 6.5 have 32 and 64 bit libraries installled on them. But up to
python 3.4 there isn't an issue with using the 32-bit libraries. Python 3.5
doesn't load a 32-bit library.. It depends on what python version is
installed on the host. So, attempt to load the 32-bit lib first and if that
fails then load the 64-bit version.


On Thu, Jun 23, 2016 at 10:26 PM, Mark Sterin <notifications@github.com>
wrote:

> In esx_service/utils/kvESX.py
> <https://github.com/vmware/docker-volume-vsphere/pull/479#discussion_r68272042>
> :
>
> > @@ -31,6 +31,7 @@
> >
> >  # VSphere lib to access ESX proprietary APIs.
> >  DISK_LIB = "/lib/libvmsnapshot.so"
> > +DISK_LIB64 = "/lib64/libvmsnapshot.so"
>
> Can you elaborate on how the loading of correct versions of esx and our
> Libs supposed to work end to end, ? Are you saying in esx 6.5 the 32 bit
> lib is missing and you use it for determine the bitness ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/479/files/00caea071825a4f36c3479c7d75ed8ba03a003bc#r68272042>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APHseKUr6boNMXno2UnazDodXX7EsQyyks5qOrqzgaJpZM4I4qrt>
> .
>